app/experimenter/experiments/api/v5/types.py

```diff
@@ -0,0 +1,39 @@
+from graphene_django.types import DjangoObjectType
+
+from experimenter.experiments.models.nimbus import (
+    NimbusBranch,
+    NimbusBucketRange,
+    NimbusExperiment,
+    NimbusIsolationGroup,
+)
+from experimenter.projects.models import Project
+
+
+class NimbusBranchType(DjangoObjectType):
+    class Meta:
+        model = NimbusBranch
+        exclude = ("id", "experiment", "nimbusexperiment")
```

Comment:
I usually do it the other way around and explicitly list fields rather than using exclude, but then I was thinking about it and I realized that makes sense for a REST API where you only care about some given subset, but for GQL you want the other way around and just expose the whole kit'n'kaboodle.

Reply:
Right, the client-side is going to request whatever subset it cares about. Since this is an internal tool it feels a bit faster on turn-around to exclude up front.